CodeWarsExercises/Codewars/CountingSheepExercise.cs

```diff
@@ -0,0 +1,42 @@
+﻿namespace CodeWarsExercises.Codewars {
+    internal class CountingSheepExercise : IExercise {
+        public string Name => "Counting sheep...";
+
+        public static int CountSheeps(bool?[] places) {
+            if (places is null)
+                throw new ArgumentNullException(nameof(places));
+
+            int counter = 0;
+
+            foreach (var sheep in places) {
+                if (!sheep.HasValue)
+                    throw new ArgumentException("There's musn't be nulls for sheeps.");
+
+                if (sheep.Value)
+                    ++counter;
+            }
+
+            return counter;
+        }
+
+        public void Run() {
+
+            Console.Write("Describe sheep places splited by space. \n" +
```

Comment:
there are possibility to use multiline strings

Reply:
Thank you for advice! If I'm right, according the documentation `multiline strings` are supported from `dotnet 7.0`. Will I break the compatibility with `dotnet 6.0` and older if I use the feature?

I'm trying to improve look of this code piece without this feature now.

Reply:
I have improved formatting of this piece. Does it looks better?

```C#
//...
        public void Run()
        {
            {
                string message =
                    "Describe sheep places splited by space. \n" +
                    "enter 'true' per place if sheep is in it's place or 'false' otherwise: ";

                Console.Write(message);
            }

            bool[] sheeps = Console.ReadLine()?.
//...
```